LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Just want to confirm this is the desired license since most things used to use Apache 2 at cLabs. 
Noticed the core repo is missing a license file so would be good to fix that too: https://github.com/mento-protocol/mento-core/

Reply:
I added a not to our backlog to get licences in order across all repos, we can merge this with MIT for now, as the decision feels a bit outside of the scope.